lib/handlebars/runtime.js

```diff
@@ -85,6 +85,23 @@ export function template(templateSpec, env) {
       return typeof current === 'function' ? current.call(context) : current;
     },
 
+    splat: function() {
+      return Utils.extend.apply(null, arguments);
```

Comment:
What is the first argument passed here? Are we ok with it being modified in all cases?